<launch>
    <node  pkg="qfy_ibvs" type="apriltag_points_node" name="apriltag_points_node" output="screen">
        <rosparam param="tag_descriptions">[
          {id: 47, size: 0.034000,frame_id: target1},
          ]
        </rosparam>
    </node>

    <node ns="camera" pkg="uvc_camera" type="uvc_camera_node" name="uvc_camera" output="screen">
        <param name="width" type="int" value="640" />
        <param name="height" type="int" value="480" />
        <param name="format" type="string" value="jpeg" />
        <param name="fps" type="int" value="60" />
        <param name="frame" type="string" value="webcam" />
        <param name="device" type="string" value="/dev/video1" />
        <param name="camera_info_url" type="string" value="file://$(find ffcalib)/config/ost_c.yaml" />
        <remap from="/camera/camera_info" to="/f_camera/camera_info"/>
    </node>

    <node pkg="image_transport" name="republish_uvc"  type="republish" args="compressed in:=/camera/image_raw _image_transport:=compressed raw out:=/f_camera/image_raw" />
    <node ns="f_camera"  pkg="image_proc" name="image_proc"  type="image_proc" />
</launch>